docs/client/AccountConsentApi.md

```diff
@@ -0,0 +1,66 @@
+# \AccountConsentApi
```

Comment:
Should this "\" be here (and some other places)?

Reply:
The forward slash... Not sure why that is being included. I can take a look.

Reply:
Here is the template for the API documentation:

```
# {{invokerPackage}}\{{classname}}{{#description}}
```
All of the API documentation should have this same problem.
Looks like we may need to specify some additional information for `invokerPackage` and/or `classname`.

Reply:
https://pureport.atlassian.net/browse/PE-1045